py3status/modules/imap.py

```diff
@@ -64,11 +67,30 @@ def check_mail(self):
 
         return response
 
+    def _connection_ssl(self):
+        connection = imaplib.IMAP4_SSL(self.imap_server,
+                                       self.port,
+                                       ssl_context=create_default_context()
```

Comment:
from python docs
> Changed in version 3.3: ssl_context parameter added.

We support python 2.6 so this will break peoples setups. Either this needs removing or you need to check we can use it.